### ISSUE TITLE
Calling PropTypes validators directly is not supported by the `prop-types` package.

### ISSUE THREAD

**Issue by fauzie811** [Exilz/react-native-render-html#6]
![simulator screen shot jul 7 2017 17 38 49](https://user-images.githubusercontent.com/318095/27954706-5deaebd0-633b-11e7-8541-28865bd38655.png)

Errors when using css inline style. E.g.:

`<h1 style="margin-top: 20px;">Header</h1>`

Using:

```
"react": "16.0.0-alpha.12",
"react-native": "0.45.1",
"react-native-render-html": "^2.0.0",
````

**Comment by fauzie811** [comment]
I don't know what happened. The error stays even after I removed the inline styling.

🤔

**Comment by Exilz** [comment]
This should be fixed in `2.1.0`
